libraries/AP_NavEKF2/AP_NavEKF2_core.h

```diff
@@ -524,6 +524,19 @@ class NavEKF2_core : public NavEKF_core_common
         uint32_t time_ms;           // measurement timestamp (msec)
     };
 
+    // Specify source of data to be used for a partial state reset
+    // Checking the availability and quality of the data source specified is the responsibility of the caller
+    enum resetDataSource {
```

Comment:
I'm really not sure about this, we create a whole bunch of enum values but then only the EXTNAV one is implemented?

I'm more and more tempted to say that we should just walk away and focus on the EKF3

Reply:
> I'm more and more tempted to say that we should just walk away and focus on the EKF3

Yes. I think you are right. It is time to move forward. I will close this one. Thank you very much for all suggestions and review.